src/main/proto/ga4gh/read_service.proto

```diff
@@ -91,6 +103,52 @@ message GetReadGroupSetRequest {
   string read_group_set_id = 1;
 }
 
+// ******************  /readgroups  *********************/
+
+// This request maps to the body of `POST /readgroups/search` as JSON.
+
+
+message SearchReadGroupsRequest {
+  // The dataset to search.
```

Comment:
I would be very careful of using `required` - please take at the following note as [required is forever](https://developers.google.com/protocol-buffers/docs/proto):

> :exclamation: **Required Is Forever** You should be very careful about marking fields as **required**. If at some point you wish to stop writing or sending a required field, it will be problematic to change the field to an optional field – old readers will consider messages without this field to be incomplete and may reject or drop them unintentionally. You should consider writing application-specific custom validation routines for your buffers instead. Some engineers at Google have come to the conclusion that using **required** does more harm than good; they prefer to use only **optional** and **repeated**. However, this view is not universal.